.appveyor.yml

```diff
@@ -0,0 +1,23 @@
+# Operating system (build VM template)
+os: Visual Studio 2015
+
+# clone directory
+clone_folder: c:\xmr-stak-cpu
+
+install:
+  - call "%VS140COMNTOOLS%\VsMSBuildCmd.bat"
```

Comment:
Is call forking the shell and than executing the bat file, if so than this is the reason for the missing compiler